client/components/credit-card-form-fields/index.jsx

```diff
@@ -92,6 +92,10 @@ export class CreditCardFormFields extends React.Component {
 		this.updateFieldValues( event.target.name, event.target.value );
 	};
 
+	addExplainer = explainerText => {
```

Comment:
Refer this comment, if we model it as suggested, we no longer need this function.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/33034/commits/25a06704d7abfa647067fcc06631d7846a6c9936.